projects/plugins/wpcomsh/custom-colors/js/color.js

```diff
@@ -439,13 +443,13 @@
 
 		// http://www.w3.org/TR/2008/REC-WCAG20-20081211/#relativeluminancedef
 		toLuminosity: function () {
-			var rgb = this.toRgb();
-			var lum = {};
-			for ( var i in rgb ) {
-				if ( ! rgb.hasOwnProperty( i ) ) {
+			const rgb = this.toRgb();
+			const lum = {};
+			for ( const i in rgb ) {
+				if ( ! Object.hasOwn( rgb, i ) ) {
```

Comment:
Similar to the above, `Object.hasOwn()` was added to most browsers in 2021 versions (Safari 15.4 was in 2022).